File: problems/1887.reduction.operations.to.make.the.array.elements.equal/Solution.go
func reductionOperations(nums []int) int {
	sort.Ints(nums)
	count := 0
	add := 0
	for i := 1; i < len(nums); i++ {
		if nums[i] != nums[i-1] {
			add++
			count += add
		} else {
			count += add
		}

	}
	return count
}

File: problems/1887.reduction.operations.to.make.the.array.elements.equal/Solution.py
from typing import List

class Solution:
    def reductionOperations(self, nums: List[int]) -> int:
        nums.sort()
        res = 0
        operations = 0

        for i in range(1, len(nums)):
            if nums[i] > nums[i - 1]:
                operations += 1
            res += operations
        return res
    
sol = Solution()
nums = [5, 3, 2]
print(sol(nums))